source/versioned-api/versioned-api.rst

```diff
@@ -169,6 +169,16 @@ the specified value is equal to the server default. Drivers MUST NOT add any
 API versioning options if the user did not specify them.
 
 
+Handshake behavior
+~~~~~~~~~~~~~~~~~~
+
+Since the legacy hello command is not part of the versioned API, drivers MUST
+NOT use this command during the initial handshake or afterwards. Instead,
+drivers MUST use the ``hello`` command exclusively. If the server does not
+support ``hello``, the server description MUST reflect this with an ``unknown``
```

Comment:
Capital ``Unknown`` for consistency with SDAM.

Reply:
Changed. Thanks!